tests/e2e/config.ts

```diff
@@ -4,7 +4,8 @@ const OUTPUT_DIR = process.env.WORKING_DIRECTORY || './tests/e2e/results';
 // add your test name here …
 const TEST_NAMES = {
     AppStartTime: 'App start time',
-    OpenSearchPage: 'Open search page TTI',
+    // ASK: Should we also update this name?
+    OpenChatFinderPage: 'Open chat finder page TTI',
```

Comment:
Yea, we should. I don't think this value is being saved anywhere (at least I couldn't find that in code)